src/SFML/Graphics/Text.cpp

```diff
@@ -231,6 +294,11 @@ void Text::draw(RenderTarget& target, RenderStates states) const
 
         states.transform *= getTransform();
         states.texture = &m_font->getTexture(m_characterSize);
+
+        // Only draw the outline if there is something to draw
+        if (m_outlineThickness > 0)
```

Comment:
Ok. I totales agree with you. But if (m_outlineThickness) is used everywhere else in the code to see if the outline is used or not. I just wanted it to be consistent. But I agree if (m_outlineThickness > 0) is more expressive.